Spanner/src/Session/CacheSessionPool.php

```diff
@@ -643,31 +649,19 @@ private function getSession(array &$data)
      */
     private function createSessions($count)
     {
-        $args = [
+        $res = $this->database->connection()->batchCreateSessions([
```

Comment:
This will need to be called in a while loop since it's possible for batchCreateSessions to return fewer than $count sessions.